projects/tuist/fixtures/ios_app_with_static_frameworks_with_resources/Project.swift

```diff
@@ -14,7 +14,12 @@ let project = Project(
                 .project(target: "A", path: "Modules/A"),
                 .project(target: "C", path: "Modules/C"),
                 .framework(path: "Prebuilt/prebuilt/PrebuiltStaticFramework.framework"),
-            ]
+            ],
```

Comment:
This was added to allow testing archiving the entire app (there was a bit code archive failure for this particular fixture)